tests/test-cases/core.yaml

```diff
@@ -1,24 +1,59 @@
+# yaml-language-server: $schema=schema.json
+
 tests:
+  - name: "test tty"
+    enabled: true
+    snapshot: false
+    tty: true
+    description: "Ensure tty is enabled."
+    workdir: "../"
+    command: "tty"
+    expect:
+      stdout:
+        - "^(/dev)"
+      stderr:
+        - "^$"
+      exit_code: 0
+
+  - name: "test non-tty"
+    enabled: true
+    snapshot: false
+    tty: false
+    description: "Ensure tty is disabled."
+    workdir: "../"
+    command: "tty"
+    expect:
+      stdout:
+        - "^not a tty"
+      stderr:
+        - "^$"
+      exit_code: 1
+
   - name: "which atmos"
     enabled: true
+    snapshot: false
     description: "Ensure atmos CLI is installed and we're using the one that was built."
     workdir: "../"
     command: "which"
     args:
       - "atmos"
     expect:
       stdout:
-        - '(build[/\\]atmos|atmos[/\\]atmos)'
+        # build/atmos is local
+        # atmos/atmos is in GitHub Actions
+        - '(build[/\\]atmos|atmos/atmos)'
       stderr:
         - "^$"
       exit_code: 0
 
   - name: "atmos"
     enabled: true
+    snapshot: true
     description: "Verify atmos CLI reports missing stacks directory."
     workdir: "../"
     command: "atmos"
     expect:
+      diff: []
```

Comment:
maybe we can do something to avoid adding empty brackets

Reply:
It's optional; you don't have to specify it, and it will default to this. I like to be explicit, since it reads well. 

"Expect no differences"
```
...
  expect:
    diff: []
```

On the otherhand, we did something like 
```
  expect:
    diff: false
```
This to me, is do not check for differences. 

In the end, my preference is we stick with the current way of expressing it.